.gitmodules

```diff
@@ -21,3 +21,6 @@
 [submodule "third_party/gpu-burn"]
 	path = third_party/gpu-burn
 	url = https://github.com/wilicc/gpu-burn.git
+[submodule "third_party/AMF"]
+	path = third_party/AMF
+	url = https://github.com/GPUOpen-LibrariesAndSDKs/AMF.git
```

Comment:
the module seems too large (GB level) and is not used in most cases
maybe move to dockerfile or makefile and clone only when building directx?

Reply:
moved to the third_party/Makefile directly